mapclassify/classifiers.py

```diff
@@ -198,7 +201,7 @@ def head_tail_breaks(values, cuts):
     """
     values = np.array(values)
     mean = np.mean(values)
-    if len(cuts) > 0 and cuts[-1] == mean:  # fix floating point issue #117
```

Comment:
Can you leave the comment here? It may not be immediately clear why we do this so the link to #117 will help.

Reply:
Sure, and I'll update the comment's wording. I deleted because it reads more like a 'to do' than a 'done'.